gradle/imglysdk.gradle

```diff
@@ -0,0 +1,42 @@
+// This file had to stay in Groovy
+// Since just converting it to Kotlin will not work
+// There needs to be a way to conditinally apply this plugin
+// But that results in missing references for `imglyConfig`
```

Comment:
so how does it work now, is the library properly excluded in foss build variant?

Reply:
Yeah, because this file is included in the build only when in proprietary build (using the same condition that was used previously).